components/backup/src/endpoint.rs

```diff
@@ -97,6 +102,8 @@ impl Task {
                 storage,
                 concurrency: req.get_concurrency(),
                 cancel: cancel.clone(),
+                is_raw_kv: req.get_is_raw_kv(),
+                cf: req.get_cf().to_string(),
```

Comment:
How about backup all cf or `default cf` when user did not pass `cf` in BakcupRequest

Reply:
I prefer default cf by default

Reply:
I agree